web-common/src/features/file-explorer/NavFile.svelte

```diff
@@ -51,36 +50,34 @@
   $: isDotFile = fileName && fileName.startsWith(".");
   $: isProtectedFile = PROTECTED_FILES.includes(filePath);
 
-  async function navigate(filePath: string) {
-    await goto(`/files${filePath}`);
-  }
-
   function handleMouseDown(e: MouseEvent) {
     if (PROTECTED_FILES.includes(filePath)) return;
     onMouseDown(e, { id, filePath, isDir: false, kind: resourceKind });
   }
 </script>
 
-<button
+<a
   aria-label="{fileName} Nav Entry"
   class="w-full h-6 group pr-2 text-left flex justify-between gap-x-1 items-center
   {isProtectedDirectory || isDotFile
-    ? 'text-gray-500'
+    ? 'text-gray-500 hover:text-gray-500'
     : 'text-gray-900 hover:text-gray-900'}
   {isCurrentFile ? 'bg-slate-100' : ''} 
   font-medium hover:bg-slate-100"
   {id}
-  on:click={() => navigate(filePath)}
+  href={`/files${filePath}`}
```

Comment:
This might be a problem for firing telemetry. We cant really add beforeNavigation since we wont have the source of the action (`left-nav` or `cta` etc)

Reply:
We should eventually be instrumenting telemetry from a single place in the code, a la https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/plugins/web/opentelemetry-instrumentation-user-interaction?ref=asserts.ai

Reply:
Merged `main` and resolved the telemetry conflict – used both `href` and `on:click` as in `DashboardCTAs.svelte`

Reply:
@briangregoryholmes, very much agree. We should consider along with this workstream: https://github.com/rilldata/rill-private-issues/issues/163